vector/v.out.lidar/v.out.lidar.html

```diff
@@ -48,8 +48,3 @@ <h2>REFERENCES</h2>
 <h2>AUTHOR</h2>
 
 Vaclav Petras
-
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
```

Comment:
Line 19, 28, 40, 47 unnecessary one empty line.

Missing  `<h2>SEE ALSO</h2>` section, maybe it is ok?

Reply:
I added the  `<h2>SEE ALSO</h2>` section.